### ISSUE TITLE
Tagging errors for "not"

### ISSUE THREAD

**Issue by muchang** [UniversalDependencies/UD_English-EWT#188]
According to the [Universal Dependencies](https://universaldependencies.org/u/pos/PART.html), the word `not` should be tagged as `PART`. 
> Negation particle: [en] not; [de] nicht

However, in EWT, there are about 164 `not` items that are labeled as `ADV`. Such as, 
```
# sent_id = weblog-juancole.com_juancole_20030911085700_ENG_20030911_085700-0015
# text = On the other, restrictions should be lifted on the mainstream Muslim League-N and Pakistan People's Party, who can defeat the MMA in fair elections if not hogtied by the secret police.
1	On	on	ADP	IN	_	3	case	3:case	_
2	the	the	DET	DT	Definite=Def|PronType=Art	3	det	3:det	_
3	other	other	ADJ	JJ	Degree=Pos	8	obl	8:obl:on	SpaceAfter=No
4	,	,	PUNCT	,	_	8	punct	8:punct	_
5	restrictions	restriction	NOUN	NNS	Number=Plur	8	nsubj:pass	8:nsubj:pass	_
6	should	should	AUX	MD	VerbForm=Fin	8	aux	8:aux	_
7	be	be	AUX	VB	VerbForm=Inf	8	aux:pass	8:aux:pass	_
8	lifted	lift	VERB	VBN	Tense=Past|VerbForm=Part|Voice=Pass	0	root	0:root	_
9	on	on	ADP	IN	_	15	case	15:case	_
10	the	the	DET	DT	Definite=Def|PronType=Art	15	det	15:det	_
11	mainstream	mainstream	ADJ	JJ	Degree=Pos	15	amod	15:amod	_
12	Muslim	Muslim	ADJ	NNP	Degree=Pos	15	amod	15:amod	_
13	League	League	PROPN	NNP	Number=Sing	15	compound	15:compound	SpaceAfter=No
14	-	-	PUNCT	HYPH	_	15	punct	15:punct	SpaceAfter=No
15	N	N	PROPN	NNP	Number=Sing	8	obl	8:obl:on|24:nsubj	_
16	and	and	CCONJ	CC	_	20	cc	20:cc	_
17	Pakistan	Pakistan	PROPN	NNP	Number=Sing	18	compound	18:compound	_
18-19	People's	_	_	_	_	_	_	_	_
18	People	People	PROPN	NNPS	Number=Plur	20	nmod:poss	20:nmod:poss	_
19	's	's	PART	POS	_	18	case	18:case	_
20	Party	Party	PROPN	NNP	Number=Sing	15	conj	8:obl:on|15:conj:and|24:nsubj	SpaceAfter=No
21	,	,	PUNCT	,	_	15	punct	15:punct	_
22	who	who	PRON	WP	PronType=Rel	24	nsubj	15:ref	_
23	can	can	AUX	MD	VerbForm=Fin	24	aux	24:aux	_
24	defeat	defeat	VERB	VB	VerbForm=Inf	15	acl:relcl	15:acl:relcl	_
25	the	the	DET	DT	Definite=Def|PronType=Art	26	det	26:det	_
26	MMA	MMA	PROPN	NNP	Number=Sing	24	obj	24:obj	_
27	in	in	ADP	IN	_	29	case	29:case	_
28	fair	fair	ADJ	JJ	Degree=Pos	29	amod	29:amod	_
29	elections	election	NOUN	NNS	Number=Plur	24	obl	24:obl:in	_
30	if	if	SCONJ	IN	_	32	mark	32:mark	_
31	not	not	ADV	RB	_	32	advmod	32:advmod	_
32	hogtied	hogtie	VERB	VBN	Tense=Past|VerbForm=Part	24	advcl	24:advcl:if	_
33	by	by	ADP	IN	_	36	case	36:case	_
34	the	the	DET	DT	Definite=Def|PronType=Art	36	det	36:det	_
35	secret	secret	ADJ	JJ	Degree=Pos	36	amod	36:amod	_
36	police	police	NOUN	NN	Number=Sing	32	obl	32:obl:by	SpaceAfter=No
37	.	.	PUNCT	.	_	8	punct	8:punct	_
``` 
Should we correct the labels from `ADV` to `PART`?

Commit: 2f5eba2

**Comment by nschneid** [comment]
It seems that in EWT, "not" is tagged as `PART` when it appears in a verb complex (should not, cannot, don't, is not, etc.). 
It is tagged as `ADV` when modifying adjectives/adverbs (though "not only" is inconsistent—sometimes ADV, sometimes PART). 

A few are tagged as ADV before an AUX or VERB where PART looks correct: "not being/AUX", "not sending/VERB". Not sure what the policy before an infinitive should be: currently "best not/ADV to think about it".

In the "X, not Y" construction it is CCONJ.

So, is the right strategy to clean up the inconsistencies and clarify the guidelines, or apply `PART` across the board?

**Comment by nschneid** [comment]
In GUM it is uniformly PART.

**Comment by amir-zeldes** [comment]
I think morphologically we can consider all of these to be the same, so I'd be OK with PART or ADV, but don't think we need to distinguish these cases. The xpos is RB, so ADV is not really a stretch, but considering the guidelines say PART and there are other corpora than EWT and GUM, I'd just as soon stick to PART.

**Comment by nschneid** [comment]
FWIW, English-ParTUT has exclusively PART, while English-PUD and English-LinES have a mixture of PART and ADV. So any policy will require revisions to multiple corpora for consistency. Of course, converting to always-PART is much easier than the other direction.

**Comment by muchang** [comment]
Thanks for the discussion, Nathan and Amir. Making them always-PART sounds good. It might be the easiest way to make the corpora be consistent without modifying the guideline.

**Comment by nschneid** [comment]
fixed in #64
